webapp/src/client/app/modules/web-client/form/web-client-form.component.scss

```diff
@@ -44,7 +44,7 @@
 
 .box {
   background-color: var(--box-background-color);
-  width: 377px;
+  width: 447px;
```

Comment:
note: Even if this change is fairly small, it should be a separate PR/commit with an appropriate description so that it can be properly tracked and logged in our changelog.